panoptes_client/collection.py

```diff
@@ -86,3 +93,34 @@ def _build_subject_list(self, subjects):
             _subjects.append(_subject_id)
 
         return _subjects
+
+    def add_default_subject(self, subject):
```

Comment:
Could you add docstrings to the new methods? (They're automatically included in the documentation.)